src/index.html

```diff
@@ -2,16 +2,176 @@
+            type="date" 
+            id="birthday" 
+            name="birthday" 
+            required
+          >
+        </label>
+
+        <label for="agreement" class="label">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            id="agreement" 
+            name="agreement" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+
+        <legend>Registration</legend>
+
+        <label for="email" class="label">
+          E-mail:
+          <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password"class="label">
+          Password:
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+          > 
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+
+        <legend>An interesting fact about you!</legend>
+
+        <label for="cats">
+          Do you love cats?
+          <input type="radio" id="cats" name="cats">
+          <span>yes</span>
+          <input type="radio" id="cat" name="cat">
+          <span>No</span>
+        </label>
+
+        <label for="color"class="label">
+          What is your favorite color?
+          <input type="color" id="color" name="color">
+        </label>
+
+        <label for="time"class="label">
+          What time do you go to bed?
+          <input 
+          type="time" 
+          id="time" 
+          name="time" 
+          step="1"
+          class="input"
+        >
+        </label>
+
+        <label for="cars"class="label">
+          What are your favorite brand of cars?
+          <select name="cars" id="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="rate"class="label">
+          How do you rate our work?
+          <input 
+            type="range" 
+            id="rate" 
+            name="rate" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label for="comments"class="label">
+          comments:
+          <textarea 
+            class="input"
+            id="comments" 
+            name="comments" 
+            minlength="10" 
+            maxlength="150"
+          >
+          </textarea>
+        </label>
+
+        <label for="recommend"class="label">
+          Would you recommend us?
+          <select name="recommend" id="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button type="submit" value="Submit"class="fieldset">Submit</button>
```

Comment:
you don't need `value` attribute here - since `value` is what will be sent to a server - button just triggers the submission, but not sending any data from itself.
And class seems redundant as well - you don't need bottom margin for a button
```suggestion
      <button type="submit">Submit</button>
```